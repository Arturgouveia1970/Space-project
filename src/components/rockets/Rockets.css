.rockets {
  width: 100%;
  display: flex;
  justify-content: center;
}

.rocket-list {
  width: 80%;
  display: flex;
  flex-direction: column;
  gap: 40px;
}

.rocket {
  display: flex;
  flex-direction: row;
  justify-content: left;
  gap: 12px;
  flex-wrap: wrap;
}

.rocket-info {
  display: flex;
  flex-direction: column;
}

.rocket-img {
  width: 80%;
  height: auto;
  box-shadow: 1px 1px 10px 1px rgb(202, 202, 202);
}

.rocket-name {
  display: flex;
  align-items: flex-start;
  justify-content: left;
  font-weight: 700;
  font-size: 1.17rem;
  margin-top: 18.72px;
}

.rocket-description {
  text-align: justify;
  font-size: 16px;
  margin: 16px 0;
}

.reserved {
  width: min-content;
  height: min-content;
  padding: 1px 4px;
  font-size: 12px;
  font-weight: 600;
  border-radius: 3px;
  background-color: rgb(86, 201, 185);
  color: white;
  text-align: center;
  vertical-align: middle;
  margin-right: 8px;
}

.reserve-btn {
  width: fit-content;
  height: 30px;
  padding: 2px 6px;
  border: none;
  background-color: rgb(40, 151, 255);
  border-radius: 4px;
  color: white;
}

@media screen and (min-width: 768px) {
  .rocket {
    flex-wrap: nowrap;
  }

  .rocket-img {
    width: 30%;
  }
}
