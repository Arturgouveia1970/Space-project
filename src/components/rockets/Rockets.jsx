import React from 'react';

const Rockets = () => (
  <div>Rockets go here!</div>
);

export default Rockets;
