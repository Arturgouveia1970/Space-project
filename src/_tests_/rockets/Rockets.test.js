import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import Rockets from '../../components/rockets/Rockets';

describe('snapshots testing', () => {
  test('snapshots for Missions component', () => {
    const rendererComponents = renderer
      .create(
        <Provider store={store}>
          <Rockets />
        </Provider>,
      )
      .toJSON();
    expect(rendererComponents).toMatchSnapshot();
  });
});
