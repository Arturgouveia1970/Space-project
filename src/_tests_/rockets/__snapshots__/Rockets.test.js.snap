// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots testing snapshots for Missions component 1`] = `
<div
  className="rockets"
>
  <ul
    className="rocket-list"
  />
</div>
`;
